src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/UpdateListingRequest.java

```diff
@@ -39,6 +40,13 @@
     @MinDuration(minMinutes = 1)
     private Duration duration;
 
+    @NotNull
+    @Min(1)
+    private Integer numberOfSessions;
```

Comment:
Same as above and in some other places too.

Reply:
Done